docs/resources/deployment-examples/example-fargate/example_config/anchor_config.yaml

```diff
@@ -150,7 +150,7 @@ app-config:
     interactiveJwtExpiration: 3600
 
     # The interactive URL where the platform server will redirect to start the SEP-24 interactive flow.
-    interactiveUrl: http://localhost:8081/sep24/interactive
+    interactive_url: http://localhost:8083/sep24/interactive
```

Comment:
Why did we change the attribute to snake case? The rest of the attributes use camel case. Won't think break things?

Reply:
I'm pretty sure it's case insensitive, either way works